deploy/kiali/kiali_cr.yaml

```diff
@@ -135,6 +135,9 @@ spec:
 #  the individual's  RBAC roles in OpenShift. Not valid for non-OpenShift environments.
 # Choose "ldap" to enable LDAP based authentication. There are additional configurations for
 #  LDAP auth strategy that are required. See below for the additional LDAP configuration section.
+# Choose "openid" to enable OpenID connect based authentication. Your cluster is required to
+#  be configured to accept the tokens issued by your IdP. There are additional required
+#  configurations for this strategy. See below for the additional OpenID configuration section.
 # When empty, its value will default to "openshift" on OpenShift and "login" on Kubernetes.
```

Comment:
remember, if we default to "token" - people will have to go through some hoops to find out how to login (do YOU know how to get a valid login token?).. so for demo purposes or people just getting started with kiali, they may not know how to login when presented with the login screen.

Reply:
If we are going to deprecate `login` then we can't use `login` as a default for more than maybe one more release.  Perhaps we should just default to `anonymous`, I mean it's about as useful as `login` and for demo's why even have a login?  Any prod install will want to use a real auth strategy.  On OS we can still default to `openshift`.

Reply:
+1 on defaulting to anonymous. Would get new users up and running faster and they won't have to worrying about obtaining any credentials.

Reply:
BTW: I would do this "deprecating the login strategy" in a separate PR so as not to make this more complicated.

Reply:
I am going to backtrack and say the default should NOT be anonymous. We need to have true auth here... so default to "token". Upstream Istio people are worried that even the demo profile comes with out of box admin/admin credentials that they don't like

Reply:
I agree and prefer deprecation to be done in another PR.
And, IMO, "token" should be the default for kubernetes. Even if we wouldn't deprecate anything, like Istio folks I don't like too much the fact of leaving the doors open with `anonymous` nor partially open with `login` with default credentials.

Reply:
@israel-hdez @jmazzitelli OK, we're in agreement to deprecate Login and LDAP in 1.19.0 (assuming this makes it into 1.19.0).  And the new default will be Token.  This will also handle the recent complaint about the admin/admin login account.  Mazz, perhaps you can let them know that we're responding in our upcoming release.

cc @lucasponce FYI about auth direction in case you have any comment/concern.